analyzers/tests/SonarAnalyzer.UnitTest/CBDE/PreservingEncodingTest.cs

```diff
@@ -18,55 +18,44 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
- extern alias csharp;
-
 using System;
 using System.Text;
-using csharp::SonarAnalyzer.CBDE;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
+using CS = SonarAnalyzer.CBDE;
```

Comment:
I think this is confusing. CS is usually used for C# but in this case is just an alias for CBDE namespace.

I prefer the initial version.